config_webrx.py

```diff
@@ -336,6 +336,36 @@
 # path to the aprs symbols repository (get it here: https://github.com/hessu/aprs-symbols)
 aprs_symbols_path = "/usr/share/aprs-symbols/png"
 
+# Uncomment the following to customize gateway beacon details reported to the aprs network
+#   Plese see Dire Wolf's documentation on PBEACON configuration for complete details:
+#   https://github.com/wb2osz/direwolf/raw/master/doc/User-Guide.pdf 
+
+# Symbol in its two-character form as specified by the APRS spec at http://www.aprs.org/symbols/symbols-new.txt
+#   Default: Receive only IGate (do not send msgs back to RF)  
+# aprs_igate_symbol = "R&"
+
+# Custom comment about igate 
+#   Default: OpenWebRX APRS gateway
+# aprs_igate_comment = "OpenWebRX APRS gateway"
+
+# Antenna Power, Height, Gain 
+#   Unspecified by default
+# Transmitter power (0 by default as only RX is supported)
+# aprs_igate_power = "0"
+# Antenna height in feet
+# aprs_igate_height = "20"
```

Comment:
please use existing setting `receiver_asl` instead of introducing a new setting.

Also: all units in APRS messages in OpenWebRX are in metric units (or converted to such), so please keep this in line.

Reply:
Missed receiver_asl, will resuse.  DireWolf uses imperial units naively, but I can convert from metric if preferred

Reply:
Checking further, it looks like APRS expects height above average terrain not sea level so receiver_asl cannot be used here.  I have updated the logic to specify aprs_igate_height in meters in the config file and convert for DireWolf

http://www.aprs.org/aprsdos-pix/phg.txt